x/derivatives/keeper/lpt.go

```diff
@@ -213,9 +225,16 @@ func (k Keeper) BurnLiquidityProviderToken(ctx sdk.Context, msg *types.MsgBurnLi
 	if err != nil {
 		panic("failed to get redeemable amount")
 	}
-	// todo send fee pool
-	_ = redeemFee
 
+	if redeemFee.IsPositive() {
```

Comment:
same
https://github.com/UnUniFi/chain/pull/403#discussion_r1134871779

Reply:
This is to prevent error when the amount is zero, since there's amount zero case, but the transaction should pass. If error is returned, transaction should fail.